src/BraintreePaymentGateway.php

```diff
@@ -129,6 +130,16 @@ public function getTransaction(string $transactionId): ?Transaction
         ))->get($transactionId);
     }
 
+    public function refundTransaction(string $transactionId, ?double $amount = null)
```

Comment:
We never use double values, all values must be float
![image](https://user-images.githubusercontent.com/1162360/230663329-2b120b9f-dd12-404a-a8e8-8ceb8aa2b66f.png)

Reply:
After:
![image](https://user-images.githubusercontent.com/1162360/230664229-07c0a5eb-ee6e-4264-bc40-afd95a2638e5.png)